setup.py

```diff
@@ -28,6 +27,9 @@
         "ipython",
         "twine",
     ],
+    'rust-backend': [
+        "rusty-rlp>=0.1.15, <0.2"
+    ]
```

Comment:
Following the example of [`eth-hash`](https://github.com/ethereum/eth-hash/blob/d05c8ac0710e2dc16aae2c8a1e406ee7cbe78871/setup.py#L28-L34)